seed/building_sync/building_sync.py

```diff
@@ -75,9 +78,7 @@ def import_file(self, source):
 
         # save element tree
         if isinstance(source, str):
-            if not os.path.isfile(source):
-                raise ParsingError("File not found: {}".format(source))
-            with open(source) as f:
+            with default_storage.open(source, mode="r+") as f:
```

Comment:
What happens if `source` is not a file / cannot be found?

Reply:
Are there times when source is not a file? Can you please give an example so I can try and see what happens and fix it.

Reply:
These lines in the original code make sure the file given by `source` exists:
```python
if not os.path.isfile(source):
    raise ParsingError("File not found: {}".format(source))
```
Could you add lines to your code that will raise the same error if the file isn't found in storage?